src/components/JustInTime/JustInTime.js

```diff
@@ -0,0 +1,57 @@
+import React, { Component } from 'react';
+import propTypes from 'prop-types';
+import './JustInTime.scss';
+
+
+class JustInTime extends Component {
+  constructor() {
+    super();
+    this.state = {
+      isOpen: null,
+    };
+  }
+
+  handleToggle(e) {
+    e.preventDefault();
+    const isClicked = this.state.isOpen;
+
+    if (!isClicked) {
+      this.setState({
+        isOpen: true,
+      });
+    } else {
+      this.setState({
+        isOpen: false,
+      });
+    }
+  }
+
+  render() {
+    const toggleState = this.state.isOpen === null || this.state.isOpen === true ? 'show' : 'close';
+    const openToggleContent = this.state.isOpen === true ? 'show' : 'close';
+    return (
+      <div className="form__row form__row--just-in-time-block">
+        <div className="form__fieldset">
+          <a
+            href={`#${toggleState}-public`}
+            aria-expanded={this.state.isOpen}
+            className={`link toggle-link ${toggleState}-link`}
+            aria-label={`click to ${toggleState}`}
+            onClick={e => this.handleToggle(e)}
+          >
+            {this.props.linkText}
+          </a>
+          <div className={`just-in-time--content just-in-time--content-${openToggleContent}`}>
+            {this.props.children}
+          </div>
+        </div>
+      </div>
+    );
+  }
+}
+
+JustInTime.propTypes = {
+  linkText: propTypes.string.isRequired,
+};
+
+export default JustInTime;
```

Comment:
should we give this an id property to use with the href so it's possible to use multiple JIT messages on a page?

Reply:
@Heleen-cr I asked this question and it was not likely to happen

Reply:
ah ok that's cool then